config/redirects.js

```diff
@@ -755,20 +755,25 @@ module.exports = [
   },
```

Comment:
There should be many more redirects here, a lot of pages were moved. For instance, the whole factors folder, so every factor page should have one.